### PR TITLE
Add documentation for tests

### PR DESCRIPTION
**Pull request by omrirh**

### CODE REVIEW

File: docs/developer-guide.MD [hunk 1]
```diff
@@ -26,3 +26,9 @@ configurators:
     - {"task_name": {"res": False, "err": str(ex)}} - on failure
 - Add the new module to [configurators_mappings.py](../openshift_day2_configurator/configurators/mappings/configurators_mappings.py)
   under `configurators_mappings` dictionary.
+
+# Tests
+#### Run day2 configurator tool tests:
+```bash
+poetry run pytest openshift_day2_configurator/tests
+```
```
Comment (rnetser) [inline docs/developer-guide.MD:34]: please add that new code should be covered by tests

**Review by rnetser — CHANGES_REQUESTED** [review]

File: docs/developer-guide.MD (hunk 1 above)
Comment (omrirh) [reply docs/developer-guide.MD:36]: done

File: docs/developer-guide.MD [hunk 2]
```diff
@@ -26,3 +26,11 @@ configurators:
     - {"task_name": {"res": False, "err": str(ex)}} - on failure
 - Add the new module to [configurators_mappings.py](../openshift_day2_configurator/configurators/mappings/configurators_mappings.py)
   under `configurators_mappings` dictionary.
+- Add tests under `openshift_day2_configurator/tests` to cover the new module.
```
Comment (rnetser) [inline docs/developer-guide.MD:29]:
```suggestion
- Add tests under `openshift_day2_configurator/tests` to cover new code.
```

(does not necessarily only new module)

**Review by rnetser — CHANGES_REQUESTED** [review]

File: docs/developer-guide.MD (hunk 2 above)
Comment (omrirh) [reply docs/developer-guide.MD]: done

**Review by rnetser — APPROVED** [review]

**Review by rnetser — APPROVED** [review]

**Pull request merged**